spec/features/merchants/items/index_spec.rb

```diff
@@ -37,13 +37,15 @@
         within("#enabled_item-#{item_4.id}") do
           expect(page).to have_button("Disable")
           click_button("Disable")
-
-        #   expect(page).to have_button("Enable")
-        #   click_button "Enable"
-        #   expect(@item_1.status).to eq("Enabled")
         end
         
         expect(current_path).to eq("/merchant/#{@merchant_2.id}/items")
-        expect(item_4.status).to eq("Disabled")
-      end
+       # expect(item_4.status).to eq("Disabled")
+
+        within("#disabled_item-#{item_4.id}") do
+            expect(page).to have_button("Enable")
+            click_button "Enable"
+        end
```

Comment:
I left a comment in the previous PR about adding a `.reload.status` to your item in your test environment and that hopefully will fix this issue!